field_audit/models.py

```diff
@@ -568,13 +573,42 @@ def delete(self, *, audit_action=AuditAction.RAISE):
         return value
 
     @validate_audit_action
-    def update(self, *args, audit_action=AuditAction.RAISE, **kw):
+    def update(self, audit_action=AuditAction.RAISE, **kw):
```

Comment:
This should have a docstring to make it clear that it will do an additional DB query to get get the current value of each field for the purposes of auditing.

Reply:
Good point. See https://github.com/dimagi/django-field-audit/pull/16